RT4 Equipment parsing

ArtificialPB opened the pull request:
Added https://github.com/powerbot/powerbot/issues/1178. If the equipment tab is not opened it will set item stack size to 1.

@Timer is it possible to parse stack sizes like in rt6? https://github.com/powerbot/powerbot/blob/rsbot-api/rt6/Equipment.java#L20

Timer commented:
@ArtificialPB I'll have to see if this hook exists in rt4, I have a feeling it doesn't.

ArtificialPB commented:
Updated itemAt(Slot) to account for new equipped items parsing, fixed small widgets issue, updated minimap parsing for Game#tileToMap(Tile) to it does not brake every other game update.

Timer commented:
@ArtificialPB can you get on slack?
